### PR TITLE
Display individual fees in Cashflow Report

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...

- renders out all fees individually for the cashflow statement report (same as P&L report)

#2197

### Approvals

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

File: centrifuge-app/src/components/Report/CashflowStatement.tsx [hunk 1]
```diff
@@ -1,10 +1,16 @@
+import { CurrencyBalance } from '@centrifuge/centrifuge-js'
```
Comment (hieronx) [inline centrifuge-app/src/components/Report/CashflowStatement.tsx:1]: Unrelated to this PR but something I noticed I forgot to update in a previous PR: could you change `Principal repayments` in the P&L report to `Principal payments` (excluding the `re`)? 🙏

**Review by hieronx — COMMENTED** [review]
It's not showing any fees for me for some reason 🤔 
Screenshot 2024-06-21 at 09 52 28

**Comment by sophialittlejohn** [comment]
> It's not showing any fees for me for some reason 🤔 Screenshot 2024-06-21 at 09 52 28

Fixed, I hadn't added the `poolFeeState` to the memo dependencies!

**Review by hieronx — APPROVED** [review]
Looks good!

**Pull request merged**